src/objects/npe01__Contacts_And_Orgs_Settings__c.object

```diff
@@ -49,9 +49,9 @@
     </fields>
     <fields>
         <fullName>Contact_Role_for_Organizational_Opps__c</fullName>
-        <description>The default Contact Role automatically assigned to the Opportunity's Primary Contact when the Opportunity's Account is an Organization Account.</description>
+        <description>The default Contact Role automatically assigned to the Opportunity's Primary Contact when the Opportunity's Account is NOT a Household, One-to-One, or Individual Account.</description>
```

Comment:
Single quotes should be encoded in this metadata file as well. You should be able to get the encoded version by refreshing the object file from within your ide.